Package.swift

```diff
@@ -0,0 +1,25 @@
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Proposer",
+    platforms: [
+        .iOS(.v8)
+    ],
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(name: "Proposer", targets: ["Proposer"])
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
+        .target(name: "Proposer", dependencies: [], path: "Proposer")
+    ],
+    swiftLanguageVersions: [.v4_2]
```

Comment:
As `swift-tools-version:5.0`, maybe we should support Swift 5?

Reply:
I think it is just a declare😂.

Reply:
Nope, that's actually a flag and will be used for Swift Version